### DISCUSSION TITLE
Create alternative flags to --force

### DISCUSSION THREAD

**Discussion by zkochan** [pnpm/pnpm#3175] [Ideas]
The `--force` flag is currently used for too many things.

- [ ] force-new-store (ref #847)

**Comment by benquarmby** [comment]
I just wanted to throw my support behind this five+ year old proposal 😅 
An alternative that has less side-effects than the current `--force` behavior would be wonderful.

In the meantime, I'd appreciate your opinion on the workaround I added to #6579.
